test/libponyc-run/private-type-as-default-argument-in-public-function/main.pony

```diff
@@ -0,0 +1,6 @@
+use lib = "lib"
+
+actor Main
+  new create(env: Env) =>
+    let p = lib.Public.apply[U8]()
+    env.out.print(p.string())
```

Comment:
```suggestion
    lib.Public.apply[U8]()
```

Having just this line should work and that way we don't print to screen if it's not needed.